packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -36,6 +36,14 @@ type BaseLinkCommonProps = {
    * @default medium
    */
   size?: 'small' | 'medium';
+  hitSlop?:
```

Comment:
add jsdoc that this is native only prop?

Reply:
Added